plugins/regex_remap/README

```diff
@@ -53,11 +54,19 @@ to turn this off use the option 'no-query-string', e.g.
    ... @pparam=maps.reg @pparam=no-query-string
 
 
-Finally, you can also include the matrix parameters in the string, using the
+You can also include the matrix parameters in the string, using the
 option 'matrix-parameters', e.g.
 
    ... @pparam=maps.reg @pparam=matrix-parameters
 
+Finally, to match on the host as well, use the option 'host', e.g. ::
+
+   ... @pparam=maps.reg @pparam=host
+
+With this enabled, the string that you will need to match will look like
+
+    //host/path?query=bar
+
 
 Note that the path to the plugin must be absolute, and by default it is
 
```

Comment:
I wonder if it's time to remove this README, now that we have good documentation on docs.trafficserver. Not necessarily for this PR, but maybe file a separate Jira/PR to just nuke this README? Duplicated docs just makes life miserable.